.babelrc

```diff
@@ -1,7 +1,6 @@
 {
   "presets": [
-    ["es2015", { "modules": false }],
+    "es2015",
```

Comment:
You can define more than one `.babelrc` file in different folders if the needs are different. So, the app folder can have its own configuration that is using tree shaking and other folders can support CommonJS.